### ISSUE TITLE
My song screwed up!

### ISSUE THREAD

**Issue by kirbyfan64** [Secretchronicles/TSC#419]
I just grabbed the latest version...and was playing...and heard NOTHING!!! Then I closed it and saw in my terminal:

    Last known SDL Error : Not an Ogg Vorbis audio stream

@datahead8888 was right when it crashed VLC...but I can't reproduce this at all outside of the game!

VLC says nothing.
ffmpeg says nothing.
libav says nothing.

Then I saw this in the ffmpeg docs (I used ffmpeg to trim the audio file):

    Note: If you omit -codec:a libvorbis then the native flac audio encoder will be used by default for the ogg/ogv output container.

*slaps self* Darn...

Long story short, I fixed it and uploaded it (again) [here](https://www.dropbox.com/s/9vm6zy7jg7og4jy/land_1_fixed.ogg?dl=0). You might want to update it...
